pkg/executables/kubectl.go

```diff
@@ -293,6 +294,18 @@ func (k *Kubectl) ListCluster(ctx context.Context) error {
 	return nil
 }
 
+func (k *Kubectl) GetNodes(ctx context.Context, kubeconfig string) ([]corev1.Node, error) {
+	params := []string{"get", "nodes", "-o", "json", "--kubeconfig", kubeconfig}
+	stdOut, err := k.Execute(ctx, params...)
```

Comment:
nit: you could use the `getObject`method for this

Reply:
will look at this when doing the unit tests